core/src/main/resources/db/changelog/changesets/2024_10_17_withdrawal_reason.yaml

```
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: "survey_assign_rename"
```

```suggestion
      id: "withdrawal_reason"
```

I assume this is copy/paste leftover- this id is a duplicate of a past changeset